jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -60,27 +95,67 @@ export default function EmailSection() {
       <div className={style.bodySection}>
         {!session.isPending &&
           session.isInitialLoadComplete &&
-          'email' in currentAccount && <p className={style.currentEmail}>{currentAccount.email}</p>}
+          'email' in currentAccount && (
+            <p className={style.currentEmail}>{currentAccount.email}</p>
+          )}
 
         {unverifiedEmail?.email &&
           !session.isPending &&
           session.isInitialLoadComplete &&
           'email' in currentAccount && (
-            <div className={style.unverifiedEmail}>
-              <Icon name='alert' />
+            <>
+              <div className={style.unverifiedEmail}>
+                <Icon name='alert' />
+                <p className={style['blurb']}>
+                  <strong>
+                    {t('Check your email ##UNVERIFIED_EMAIL##. ').replace(
+                      '##UNVERIFIED_EMAIL##',
+                      unverifiedEmail.email
+                    )}
+                  </strong>
+
+                  {t(
+                    'A verification link has been sent to confirm your ownership. Once confirmed, this address will replace ##UNVERIFIED_EMAIL##'
+                  ).replace('##UNVERIFIED_EMAIL##', currentAccount.email)}
+                </p>
+              </div>
 
-              <p className={style['blurb']}>
-                <strong>
-                  {t('Check your email ##UNVERIFIED_EMAIL##. ').replace('##UNVERIFIED_EMAIL##', unverifiedEmail.email)}
-                </strong>
+              <div className={style.editEmail}>
+                <Button
+                  label='Resend'
+                  size='m'
+                  color='blue'
+                  type='frame'
+                  onClick={resendNewUserEmail.bind(
+                    resendNewUserEmail,
+                    unverifiedEmail.email
+                  )}
+                />
+                <Button
+                  label='Remove'
+                  size='m'
+                  color='red'
+                  type='frame'
+                  onClick={deleteNewUserEmail}
+                />
+              </div>
 
-                {t('A verification link has been sent to confirm your ownership. Once confirmed, this address will replace ##UNVERIFIED_EMAIL##').replace('##UNVERIFIED_EMAIL##', currentAccount.email)}
-              </p>
-            </div>
+              {email.refreshedEmail &&
+                <label>
```

Comment:
Please run through prettier, I'm seeing diffs when I apply it. I'm not sure how to enforce this on CI without running prettier on all code which IMO we should do so as to respect your time that I would rather have spent coding than doing things a bot could do.